content/documentation/rpc/wallet/multisig/create_trusted_dealer_key_package.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: wallet/multisig/createTrustedDealerKeyPackage
+description: RPC Wallet/Multisig | Iron Fish Documentation
+---
+
+Defines the structure of a multisig account. The result is a set of accounts to import by each participant using the RPC [wallet/importAccount](/developers/documentation/rpc/wallet/import_account). It is the responsibility of the dealer to securely distribute the accounts to the participants.
```

Comment:
I don't know if 'defines the structure' is how I'd describe what trusted dealer does. Maybe 'generates signing shares for' or 'generates key packages for'?

Reply:
Agree with @hughy's comment. This should probably indicate:
* Splits the `ask` and generates secret shares for each participant
* Includes a public key package with a group verifying key and participant verifying share
* Returns an encrypted account import

Reply:
Just updated, lmk what you think